.github/workflows/build-test.yml

```diff
@@ -12,7 +12,7 @@ on:
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   # This job builds & runs iceoryx tests in ubuntu-18.04
-  ubuntu-iceoryx-meta:
```

Comment:
For ubuntu we have to add 2 targets:
- does the build script and the out of tree build work
- does the cmake iceoryx_meta build work.

Even if this is done implicitly in the build script I would prefer also an explicit cmake only target since the build script can change over time and then maybe we forget to test this.

Reply:
The build script use already the iceoryx_meta as single point to build iceoryx, with the script are the most variants of iceoryx meta tested like buildall, coverage etc. I do not see what i need to test in the extra build job on iceoryx_meta then. Can you give me a hint?